strings/0004/remove-repeated-chars_francine.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+char *findUnique(char[]);
+
+int main()
+{
+    char array[100];
+    char *uniqueArray;
+
+    //ask user to enter string
+    printf("Enter string: ");
+    gets(array);
+    uniqueArray = findUnique(array);
+    printf("%s", uniqueArray);
+    return 0;
+}
+
+char* findUnique(char array[])
+{
+    int length, i, j, k = 0, count = 0;
+    char *newArray;
+    newArray = (char*) malloc(100 * sizeof(char));
+
+    // get the length of the string
+    length = strlen(array);
+
+    for (i = 0; i < length; i++) {
```

Comment:
You can also improve the efficiency of the loop by skipping the letters that were already compared.